pages/common/node.md

```diff
@@ -18,3 +18,7 @@
 - Evaluate and print result, useful to see node's dependencies versions:
 
 `node -p "{{process.versions}}"`
+
+- Enable Node's inspector agent and pause execution until a debugger is connected. Source code is fully compiled:
+
+`node --no-lazy --inspect-brk {{file}}.js`
```

Comment:
This would improve consistency with other pages :-)
```suggestion
`node --no-lazy --inspect-brk {{path/to/file.js}}`
```

Also, files don't necessarily _have_ to end in `.js` :P

Reply:
Would you accept the same changes to the other file path reference? If so i'll update the document for consistency. 

I agree with the recommended changes. My commit used `{{file}}.js` to remain consistent with the document. Reason being, since its the [accepted (prod)](https://github.com/tldr-pages/tldr/commit/856c3d1535bfa95fdd757937c3d9911bef292d8c) approach - reuse what's there. Yes, no filename is required e.g. `node {{file}}` is accurate.

Reply:
See. Latest commit.